config/samples/druid_v1alpha1_etcd.yaml

```diff
@@ -62,17 +62,17 @@ spec:
   sharedConfig:
     autoCompactionMode: periodic
     autoCompactionRetention: "30m"
-  schedulingConstraints:
-    affinity: {}
-    topologySpreadConstraints:
-    - maxSkew: 1
-      topologyKey: topology.kubernetes.io/zone
-      whenUnsatisfiable: DoNotSchedule
-      labelSelector:
-        matchLabels:
-          app: etcd-statefulset
+  # schedulingConstraints:
+    # affinity: {}
+    # topologySpreadConstraints:
+    # - maxSkew: 1
+    #   topologyKey: topology.kubernetes.io/zone
+    #   whenUnsatisfiable: DoNotSchedule
+    #   labelSelector:
+    #     matchLabels:
+    #       app: etcd-statefulset
 
-  replicas: 3
+  replicas: 1
```

Comment:
IMO, `spec.replicas` should be `3` in sample config yaml file as it shows we do support multi-node which implies support for single replica as well.

Reply:
You're right, @ishan16696. I'll make the appropriate change and change the replicas back to 3.

Reply:
I've changed the `spec.replicas` field to `3` in this sample config to show we support multi-node.